RELEASE.md

```diff
@@ -15,6 +15,7 @@
  * [OND211-1714] Implement intent cleaning endpoints.
  * [OND211-1714] Implement entity_type cleaning endpoints.
  * [OND211-1714] Implement endpoints to add new training phrases to intents.
+ * [OND211-1714] Add initiation protocol into train agent endpoint
```

Comment:
Careful.
The release 1.1.0 is already out.

All new features will be gathered together into a new version that we can track together with the `client`.
This version would be either 1.2.0 (if we were breaking backwards compatibility it would have been 2.0.0).